src/router.ts

```diff
@@ -18,6 +21,15 @@ export const router = createRouter({
     },
     // Account Management
     { path: '/account-management', component: AccountManagement },
+    // Handle Action
+    {
+      path: '/action',
+      component: HandleAction,
```

Comment:
qq: why is this named generically? will we use the HandleAction component for other actions too?

Reply:
Yes, it will be used for other actions too, like reset password.